Assets/LeapMotionModules/GraphicRenderer/Resources/GraphicRenderer.cginc

```diff
@@ -2,4 +2,4 @@
 
 // This line is auto-generated, do not modify!
 // Maximum graphic count can be changed in the preferences dialog.
-#define GRAPHIC_MAX 144
+#define GRAPHIC_MAX 48
```

Comment:
Yeah, DX9 apparently has a very small amount of registers that can hold data :( Doesn't stop you from going much higher than this as long as you don't want to support dx9